control-plane-operator/hostedclusterconfigoperator/controllers/resources/olm/params.go

```diff
@@ -0,0 +1,29 @@
+package olm
+
+import (
+	"strings"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+)
+
+type OperatorLifecycleManagerParams struct {
+	CertifiedOperatorsImage string
+	CommunityOperatorsImage string
+	RedHatMarketplaceImage  string
+	RedHatOperatorsImage    string
+	OLMCatalogPlacement     string
+}
+
+func NewOperatorLifecycleManagerParams(hcp *hyperv1.HostedControlPlane, releaseVersion string) *OperatorLifecycleManagerParams {
+	tag := strings.Join(strings.Split(releaseVersion, ".")[:2], ".")
```

Comment:
Interesting, I thought this wouldn't work for versions that are not GA yet, but it does, there are 4.11 images already